index.js

```diff
@@ -407,6 +407,14 @@ module.exports = {
 			},
 		],
 		'max-depth': 'warn',
+		'max-len': [
+			'warn',
+			200,
+		],
```

Comment:
Needs the `ignoreComments: true` option. (unclear to me whether it is the default or not)

Reply:
Ignored URLs as well